jsonschema2pojo-ant/src/main/java/org/jsonschema2pojo/ant/Jsonschema2PojoTask.java

```diff
@@ -71,6 +71,8 @@ public class Jsonschema2PojoTask extends Task implements GenerationConfig {
 
     private boolean includeTypeInfo = false;
 
+    private boolean useNestedClasses = false;
```

Comment:
The proper full name would be `useStaticNestedClasses`, but since it is rather long, so I have shortened it.